README.md

```diff
@@ -1,48 +1,47 @@
 <p align="center">
   <a href="https://amplication.com" target="_blank">
-    <img width="344" height="70" src="https://amplication.com/assets/amplication-logo.svg" alt="amplication  Logo"></a>
+    <img alt="amplication-logo" height="70" alt="Amplication Logo" src="https://amplication.com/assets/amplication-logo.svg"/>
```

Comment:
@Leeyaacov please prepare a logo that will work both on dark mode and light mode

Reply:
You can find a purple version of our logo in Figma>social

![image](https://user-images.githubusercontent.com/65485193/103199476-9b9de080-48f3-11eb-8990-2ace0f8b8bb0.png)

Reply:
Would it be okay if we'll put the logo only leaving the title as text?

Reply:
No, because the Github font is not the same as our font.
If you like to use only our logo symbol you can, but not with text saying 'amplicatoin' next to it